src/ggrc/converters/query_helper.py

```
@@ -92,7 +92,7 @@ def _find_child_type(self, expr, child_type):
     """Search for child_type recursively down the query expression."""
     if child_type:
       return child_type
-    left, oper, right = expr.get("left"), expr.get("op"), expr.get("right")
+    left, oper, right = expr.get("left"), expr.get("op", {}), expr.get("right")
```

Sorry, what is the valid case where we do not have an operator for a given expression?
Depending on the case, it might be better to force operators in all expressions instead of handling special cases like this here, and if there is an expression without an operator we might want to throw an exception rather than accepting it like this.